build/common.project.props

```diff
@@ -71,6 +71,12 @@
     <MSBuildTreatWarningsAsErrors Condition=" '$(MSBuildTreatWarningsAsErrors)' == '' And '$(IsOfficialBuild)' != 'true' ">true</MSBuildTreatWarningsAsErrors>
   </PropertyGroup>
 
+  <!-- Write out .XML files for projects that will be packed. -->
+  <PropertyGroup Condition=" '$(PackProject)' == 'true' ">
+    <GenerateDocumentationFile Condition=" '$(GenerateDocumentationFile)' == '' ">true</GenerateDocumentationFile>
+    <DocumentationFile Condition=" '$(DocumentationFile)' == '' AND '$(GenerateDocumentationFile)' == 'true' AND '$(IsNetCoreProject)' != 'true' ">$(OutputPath)\$(AssemblyName).xml</DocumentationFile>
+  </PropertyGroup>
```

Comment:
As you wrote in another comment, the condition on the group will always evaluate to false, because common props get evaluated BEFORE the project's own properties, which is why this was previously in the targets file, since common targets get evaluated after the project.

Therefore, this code here is dead code, and should be removed.  If the repo works as expected (projects are generating xml doc files, and we're getting warnings about missing public methods without docs), then we don't need the config at all.